calyx-opt/src/default_passes.rs

```diff
@@ -66,6 +64,8 @@ impl PassManager {
 
         // Disabled by default
         pm.register_pass::<UnrollBounded>()?;
+        pm.register_pass::<MergeStaticPar>()?;
+        pm.register_pass::<StaticParConv>()?;
```

Comment:
Oh, I see! This PR doesn't get rid of these passes yet. In that case, remember to remove them from the default compilation pipeline!